src/ServiceControl.LicenseManagement/LicenseManager.cs

```diff
@@ -107,6 +108,13 @@ static string GetMachineLevelLicenseLocation()
             return LicenseFileLocationResolver.GetPathFor(Environment.SpecialFolder.CommonApplicationData);
         }
 
+        static bool MustBeNonTrialLicenseForSetup()
+        {
+            var isDockerEnvironmentVariable = Environment.GetEnvironmentVariable("SERVICECONTROL_NO_TRIAL");
+
+            return string.Equals("true", isDockerEnvironmentVariable, StringComparison.InvariantCultureIgnoreCase);
```

Comment:
Is the content relevant or just the existence? If the content is relevant and needs to represent a boolean would it then make sense to use https://docs.microsoft.com/en-us/dotnet/api/system.boolean.tryparse?view=net-5.0 instead?